<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiDevices</class>
 <widget class="QDialog" name="MidiDevices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI ports Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {background-color:#333;}
QWidget {color:#ddd;}

QComboBox {background-color:#4A4A4A;}
QComboBox QAbstractItemView {background-color:#444;}
QComboBox:hover{background-color:#405080;}
QComboBox:!enabled{background-color:#202020;color:#808080;}

QPushButton{background-color:#444;color:#EEE;border:1px solid #888; border-radius:4px;}
QPushButton:!enabled{background-color:#202020;color:#808080}
QPushButton:hover{background-color:#405080;}
QPushButton:pressed {background:#505020;border:1px inset #222;}
QSpinBox:!enabled {color:#777;}
QDoubleSpinBox:!enabled {color:#777;}
QLabel {background-color: rgba(0, 0, 0, 0);}
QLabel:!enabled {color:#888;}
QLineEdit {background-color:#333;color:#EEE;border:1px solid #888;}
QLineEdit:hover{background-color:#446;}
QLineEdit:!enabled {background-color:#202020;color:#888;}
QMessageBox QPushButton {width:80px;height:20px;}

QGroupBox {border: 1px solid #555; }</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="validateButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="throughInComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MIDI Bridge (MIDI Out)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MIDI Bridge (MIDI In)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="throughOutComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>External mapping (MIDI In)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="extInComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="extOutComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>External mapping (MIDI Out)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>KB2D (MIDI In Feedback)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="kbOutComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="kbInComboBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>KB2D (MIDI Out Commands)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="updateAllButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Update all MIDI devices</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
